barretenberg/cpp/src/barretenberg/eccvm/eccvm_prover.cpp

```diff
@@ -104,72 +105,74 @@ void ECCVMProver::execute_relation_check_rounds()
+ * @brief Produce a univariate opening claim for the sumcheck multivariate evalutions and a batched univariate claim
+ * for the transcript polynomials (for the Translator consistency check). Reduce the two opening claims to a single one
+ * via Shplonk and produce an opening proof with the univariate PCS of choice (IPA when operating on Grumpkin).
+ * @details See https://hackmd.io/dlf9xEwhTQyE3hiGbq4FsA?view for a complete description of the unrolled ZeroMorph
+ * protocol.
  *
- * */
+ */
 void ECCVMProver::execute_pcs_rounds()
 {
     using Curve = typename Flavor::Curve;
     using ZeroMorph = ZeroMorphProver_<Curve>;
-    auto prover_opening_claim = ZeroMorph::prove(key->polynomials.get_unshifted(),
-                                                 key->polynomials.get_to_be_shifted(),
-                                                 sumcheck_output.claimed_evaluations.get_unshifted(),
-                                                 sumcheck_output.claimed_evaluations.get_shifted(),
-                                                 sumcheck_output.challenge,
-                                                 commitment_key,
-                                                 transcript);
-    PCS::compute_opening_proof(
-        commitment_key, prover_opening_claim.opening_pair, prover_opening_claim.polynomial, transcript);
-}
+    using Shplonk = ShplonkProver_<Curve>;
+    using OpeningClaim = ProverOpeningClaim<Curve>;
+
+    // Execute the ZeroMorph protocol to produce a univariate opening claim for the multilinear evaluations produced by
+    // Sumcheck
+    auto multivariate_to_univariate_opening_claim =
+        ZeroMorph::prove(key->polynomials.get_unshifted(),
+                         key->polynomials.get_to_be_shifted(),
+                         sumcheck_output.claimed_evaluations.get_unshifted(),
+                         sumcheck_output.claimed_evaluations.get_shifted(),
+                         sumcheck_output.challenge,
+                         commitment_key,
+                         transcript);
+
+    // Batch open the transcript polynomials as univariates for Translator consistency check. Since IPA cannot
+    // currently handle polynomials for which the latter half of the coefficients are 0, we hackily
+    // batch the constant polynomial 1 in with the 5 transcript polynomials.
 
-/**
- * @brief Batch open the transcript polynomials as univariates for Translator consistency check
- * TODO(#768): Find a better way to do this. See issue for details.
- *
- * @tparam Flavor
- */
-void ECCVMProver::execute_transcript_consistency_univariate_opening_round()
-{
-    // Since IPA cannot currently handle polynomials for which the latter half of the coefficients are 0, we hackily
-    // batch the constant polynomial 1 in with the 5 transcript polynomials. See issue #768 for more details.
     Polynomial hack(key->circuit_size);
```

Comment:
I know you didn't introduce this but do we have an issue open for this? If not we should make one to ensure we don't lose track of it.

Reply:
yup we do, i think i removed the comment that mentions it, adding it back